engine/order_manager.go

```diff
@@ -727,22 +716,19 @@ func (m *OrderManager) processOrders() {
 }
 
 func (m *OrderManager) processMatchingOrders(exch exchange.IBotExchange, orders []order.Detail, requiresProcessing map[string]bool, wg *sync.WaitGroup) {
-	defer func() {
-		if wg != nil {
-			wg.Done()
-		}
-	}()
 	for x := range orders {
-		if time.Since(orders[x].LastUpdated) < time.Minute {
+		if time.Since(orders[x].LastUpdated) < time.Minute ||
+			!requiresProcessing[orders[x].InternalOrderID] {
```

Comment:
Seems this bool check can be removed since you now delete any false ones on line 705

Reply:
🦅